### ISSUE TITLE
glutin refuses to compile on OS X

### ISSUE THREAD

**Issue by InfiniteDarkness** [tomaka/glutin#256]
```
~/.cargo/git/checkouts/glutin-666c1766c69e6584/master/src/cocoa/mod.rs:145:36: 145:82 error: mismatched types:
 expected `*mut libc::types::common::c95::c_void`,
    found `*mut libc::types::common::c95::c_void`
 (expected enum `libc::types::common::c95::c_void`,
    found a different enum `libc::types::common::c95::c_void`) [E0308]
```

Is this a compiler bug or something?

**Comment by tomaka** [comment]
Oh I totally forgot about cocoa.

I *think* https://github.com/servo/rust-core-foundation/pull/57 will fix it.

**Comment by InfiniteDarkness** [comment]
Just tried that. Nope. I still get the same error.

**Comment by tomaka** [comment]
Then you'd also need https://github.com/servo/rust-cocoa/pull/61

**Comment by InfiniteDarkness** [comment]
Yes! That works. Thanks.

**Comment by bjz** [comment]
Just merged servo/rust-cocoa#61 - seems to be fixed.
